common/app/model/content.scala

```diff
@@ -82,6 +83,7 @@ final case class Content(
   lazy val isHeroic = HeroicTemplateSwitch.isSwitchedOn && tags.isLabourLiverpoolSeries
   lazy val isImmersive = fields.displayHint.contains("immersive") || isImmersiveGallery || tags.isUSMinuteSeries || isHeroic
   lazy val isAdvertisementFeature: Boolean = tags.tags.exists{ tag => tag.id == "tone/advertisement-features" }
+  lazy val campaigns: List[Campaign] = targeting.CampaignAgent.getCampaignsForTags(tags.tags.map(_.id))
```

Comment:
dom came and discussed this and other issues on friday, basically the limits don't have to be restrictive at this stage, just they have to exist so that people have to make a conscious decision to remove them.
just take whatever you expect to use in the next month or two and multiply them all by 3, so something like 12 rules with 12 lacking/required and 12 tags over 30 campaigns.

Think of it more like a switch that will expire when it's used too much seriously, only rather than being a time based switch it's a usage based one.